README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center"><img src="resources/icon.png" width="100"></p>
+
+# Dimensions plugin for Craft CMS
+
+Replicate the Craft Commerce Dimensions fields as a standalone field type.
+
+![Field input](resources/input.png)
+
+![Settings](resources/settings.png)
+
+## Requirements
+
+This plugin requires Craft CMS 3.0.0-RC1 or later.
+
+## Installation
+
+To install the plugin, follow these instructions.
+
+1. Open your terminal and go to your Craft project:
+
+        cd /path/to/project
+
+2. Then tell Composer to load the plugin:
+
+        composer require viget/craft-dimensions
+
+3. In the Control Panel, go to Settings → Plugins and click the “Install” button for Dimensions.
```

Comment:
Eh, TBH if someone knows how to use terminal, they probably know what the command is and how to find it. Don't want to confuse a non-technical user.

Reply:
After additional though, we should probably completely rewrite these installation instructions, especially since these plugins are in the plugin store

Good example: https://github.com/craftcms/contact-form#installation